### PR TITLE
Added hr_HR locale

### PR DESCRIPTION
**Pull request by toniperic**

### CODE REVIEW

File: src/Faker/Provider/hr_HR/Address.php [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+namespace Faker\Provider\hr_HR;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $streetPrefix = array(
+        'Trg', 'Ulica', 'Avenija'
+    );
+
+    protected static $street = array(
+        'Ante Starčevića', 'bana Jelačića', 'Ive Andrića', 'Gojka Šuška', 'Vladimira Nazora', 'Augusta Cesareca',
```
Comment (fzaninotto) [inline src/Faker/Provider/hr_HR/Address.php]: Too small set of data. Faker needs to provide random street names, not the same names avery 10 calls.

File: src/Faker/Provider/hr_HR/Address.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    protected static $addressFormats = array(
+        '{{streetAddress}}, {{postcode}} {{city}}',
+    );
+
+    protected static $cityFormats = array(
+        '{{cityName}}',
+    );
+
+    protected static $cityNames = array(
+        'Zagreb', 'Split', 'Rijeka', 'Osijek', 'Varaždin', 'Dubrovnik', 'Vinkovci', 'Županja', 'Ogulin', 'Pula',
```
Comment (fzaninotto) [inline src/Faker/Provider/hr_HR/Address.php]: Could you please add more data?

File: src/Faker/Provider/hr_HR/Person.php [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+namespace Faker\Provider\hr_HR;
+
+class Person extends \Faker\Provider\Person
+{
+    /**
+     * @link http://www.behindthename.com/top/lists/croatia/2009
+     */
+    protected static $firstNameMale = array(
+        'Luka', 'Ivan', 'Marko', 'David', 'Filip', 'Josip', 'Petar', 'Karlo', 'Matej', 'Ivano', 'Roko', 'Leon',
```
Comment (fzaninotto) [inline src/Faker/Provider/hr_HR/Person.php]: Please sort the names by alphabetical order. Easier to maintain. Same for other name lists.

**Comment by toniperic** [comment]
Sorry for the wait, was really busy. Added more data and sorted alphabetically, as requested. Hope it's alright now.

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**